tests/scripts/local_monitoring_tests/rest_apis/test_bulkAPI.py

```diff
@@ -104,3 +99,127 @@ def test_bulk_post_request(cluster_type, bulk_request_payload, expected_job_id_p
         if job_id_present:
             validate_job_status(response.json()["job_id"], URL, caplog)
 
+
+@pytest.mark.test_bulk_api_ros
+@pytest.mark.parametrize("start, end, expected_error", [
```

Comment:
@khansaad Can you update the test documentation with the test scenarios. Also add a description in the PR.